components/PageLayout/index.tsx

```diff
@@ -3,14 +3,15 @@ import Head from 'next/head';
 import Router from 'next/router';
 import { Main, Container } from './PageLayout.styled';
 import { useModalContext, MODAL_TYPES } from '../Provider';
-import { WithdrawModal } from '../Modal';
+import { WithdrawModal, ClaimBalanceBanner } from '../Modal';
 
 type Props = {
   title: string;
   children: ReactNode;
+  bannerSpacing?: boolean;
```

Comment:
I just realized - instead of doing this, we should just have the `Banner` component rendered in `my-nfts/[chainAccount].tsx`. Then we won't need to conditionally render based on the pathname as well and we wouldn't need to updating the margin-top of `PageLayout`'s container.